### PR TITLE
PUBDEV-7655: Add documentation for `gainslift_bins`

### PR DESCRIPTION
**Pull request by hannah-tillman**
Added `gainslift_bins` to the parameter appendix
Added example to KS Metric in Python docs
Added examples to `gainslift_bins` in Python docs

### CODE REVIEW

File: h2o-docs/src/product/data-science/algo-params/gainslift_bins.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Related Parameters
+~~~~~~~~~~~~~~~~~~
+
+- None
+
+Example
+~~~~~~~
+
+.. tabs::
```
Comment (angela0xdata) [inline h2o-docs/src/product/data-science/algo-params/gainslift_bins.rst:22]: The R and Python examples should always match each other. And the values returned should also match in R and Python.

File: h2o-docs/src/product/data-science/algo-params/gainslift_bins.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
+``gainslift_bins``
+------------------
+
+- Available in: GBM, DRF, Naïve-Bayes, XGBoost
```
Comment (angela0xdata) [inline h2o-docs/src/product/data-science/algo-params/gainslift_bins.rst:4]: You need to link to this topic in each of the supported algo chapters (as in below)

Screen Shot 2020-07-02 at 11 36 19 AM

**Comment by Pscheidl** [comment]
Should I review this just now, or wait @hannah-tillman ?

**Comment by hannah-tillman** [comment]
> Should I review this just now, or wait @hannah-tillman ?

@Pscheidl review now please :)

**Pull request merged**